its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1047,16 +1128,19 @@ void start(TestInfo info) {
     void stop() {
       adminWsClient.settings().reset(new ResetRequest().setKeys(singletonList("sonar.java.file.suffixes")));
       try {
-        engine.stop(true);
+        engine.stop();
       } catch (Exception e) {
         // Ignore
       }
     }
 
     @Test
     @Tag(HOTSPOT_FEATURE_DISABLED)
+    @OnlyOnSonarQube(from = "9.7")
```

Comment:
What has changed for not supporting older SQ versions?

Reply:
addressed in [4989ca3](https://github.com/SonarSource/sonarlint-core/pull/799/commits/4989ca37baf60cf950245a99149ed3079780e0fa)